ui-tests/cypress/support/pageObjects/content/management/AddPage.js

```diff
@@ -3,9 +3,11 @@ import Content from "../../app/Content.js";
 import AppPage from "../../app/AppPage.js";
 import ManagementPage from "./ManagementPage";
 import DropDownButton from "./DropDownButton";
+import ContentWidgetConfigPage from "../../pages/designer/widgetconfigs/ContentWidgetConfigPage";
 
 export default class AddPage extends Content {
 
+  fromContentWidgetConfig = false;
```

Comment:
Page objects should be stateless; state information should be rather kept in the test script

Reply:
hmm I'll find a workaround on this

Reply:
should be okay now @g-di-martino